students/sikorski_marcin/Lesson_1_-_Python_basics.py

```diff
@@ -0,0 +1,128 @@
+#1 Sum of three numbers
+
+a = int(input())
+b = int(input())
+c = int(input())
+print(a+b+c)
+
+#2 Area of right-angled triangle
+a = int(input())
+b = int(input())
+print(0.5*a*b)
+
+#3 Hello, Harry!
+
+a = input()
+print("Hello, "+ a + "!")
+
+#4 Apple sharing
+a = int(input())
+b = int(input())
+print(b//a)
+print(b%a)
+
+#5 Previous and next
+a = int(input())
+print('The next number for the number ' + str(a)  + " is " + str(a+1) + ".")
+print('The previous number for the number ' + str(a)  + " is " + str(a-1) + ".")
+
+#6 School desks
+a = int(input())
+b = int(input())
+c = int(input())
+
+l = [a,b,c]
+sum = 0
+for x in l:
+    if x%2 != 0:
+        z = ((x+1)/2)
+        sum += z
+    else:
+        z = (x/2)
+        sum += z
+print(sum)
+
+#---------------------------
+#PART 2
+#---------------------------
+
+#1 Last digit of integer
+a = int(input())
+print(a%10)
+
+#2 Tens digit
+a = str(input())
+print(a[-2])
+
+#3 Sum of digits
+a = str(input())
+changeToString = str(a)
+sum = 0
+for eachNumber in changeToString:
+    sum += int(eachNumber)
+print(sum)
+
+#4 Fractional part
+import math
+a = float(input())
+frac, whole = math.modf(a)
+print(frac)
+
+#5 First digit after decimal point
+import math
+a = float(input())
+fraction = math.modf(a)
+fractionFirstElement = str(fraction[0])
```

Comment:
zrób to bez konwersji na string. Da się to też prosto zrobić bez wykorzystywania funkcji modf.